x/perpetual/keeper/query_open_estimation.go

```diff
@@ -128,6 +128,9 @@ func (k Keeper) OpenEstimation(goCtx context.Context, req *types.QueryOpenEstima
 		fundingRate = pool.FundingRate
 	}
 
+	borrowFee := borrowInterestRate.MulInt(leveragedAmount.Sub(collateralAmountInBaseCurrency.Amount))
+	fundingFee := fundingRate.MulInt(leveragedAmount)
```

Comment:
@amityadav0 funding fee value is independent from position direction, indeed if it -ve/+ve it will affect the collection/distribution funding fee process but the funding fee in itself remains the same format

Reply:
@cryptokage1996 same question as @amityadav0 asked earlier:
> what is borrowFee here ? is it borrow fees per year ? could you please add a comment ?
same for fundingFee is it to provide an estimation APR or per block basis of an amount that will be paid

Reply:
`fundingFee` seems to return an estimated amount that will be paid per block, in that case indeed we need to account for the position direction LONG or SHORT as follows:

```
| funding rate   |   direction |  funding fee                        | 
------------------------------------------------------
| +ve                 |   LONG     |  amount to pay per block   |
| +ve                 |   SHORT   |  amount to earn per block |
| -ve                 |   LONG     |  amount to earn per block   |
| -ve                 |   SHORT     |  amount to pay per block   |
```

@cryptokage1996 please update the code so the `fundingFee` returns a positive value when the position must pay the funding fee or a negative amount when the position earns the fee.

cc @cryptokage1996 @amityadav0 @Wgil

Reply:
@cryptokage1996 please review my changes here https://github.com/elys-network/elys/pull/810/commits/c0890a71d1a0ec3b3eda43d684e0eee9a123031e to apply changes mentioned above